api/test/integration/conftest.py

```diff
@@ -116,13 +119,20 @@ def build_and_up(env_mode: str, interval: int = 10, interval_build_env: int = 10
     os.makedirs(test_logs_path, exist_ok=True)
     with open(docker_log_path, mode='w') as f_docker:
         while values_build_env['retries'] < values_build_env['max_retries']:
-            if build:
+            if build and not build_managers_only:
                 current_process = subprocess.Popen(["docker", "compose", "--profile", env_mode,
                     "build", "--build-arg", f"WAZUH_BRANCH={current_branch}", 
                     "--build-arg", f"ENV_MODE={env_mode}",
                     "--no-cache"],
                     stdout=f_docker, stderr=subprocess.STDOUT, universal_newlines=True)
                 current_process.wait()
+            if build_managers_only:
+                current_process = subprocess.Popen(["docker", "compose", "--profile", env_mode,
+                    "build", "wazuh-master", "wazuh-worker1", "wazuh-worker2",
```

Comment:
We could create a `managers` profile in the docker compose file and build the services corresponding to that profile instead of hard-coding the names (which would break if they are changed, or wouldn't work if we add a `wazuh-worker3`)